StreamChatUITestsAppUITests/Tests/PushNotification_Tests.swift

```diff
@@ -6,125 +6,245 @@ import XCTest
 
 final class PushNotification_Tests: StreamTestCase {
     
-    func test_pushNotificationOnNewMessage() throws {
-        linkToScenario(withId: 27)
-        
+    let sender = "Han Solo"
+    let message = "How are you? 🙂"
+    
+    override func setUpWithError() throws {
         try XCTSkipIf(ProcessInfo().operatingSystemVersion.majorVersion < 14,
-                      "Remote push notifications do not work on iOS < 14")
+                      "Push notifications infra does not work on iOS < 14")
+        try super.setUpWithError()
+    }
+    
+    func test_goToBackgroundFromMessageList_and_tapOnPushNotification() throws {
+        linkToScenario(withId: 95)
         
-        let sender = "Han Solo"
-        let message = "hey user#1 👋"
-
-        GIVEN("user opens the channel") {
-            userRobot
-                .login()
-                .openChannel()
+        GIVEN("user goes to message list") {
+            userRobot.login().openChannel()
         }
-        AND("user goes to background") {
-            deviceRobot.moveApplication(to: .background)
+        checkHappyPath(message: message, sender: sender)
+    }
+    
+    func test_goToBackgroundFromChannelList_and_tapOnPushNotification() throws {
+        linkToScenario(withId: 291)
+        
+        throw XCTSkip("[CIS-2164] The test app is not yet ready for this test")
+
+        GIVEN("user goes to channel list") {
+            userRobot.login()
         }
-        WHEN("participant sends a message") {
-            participantRobot.wait(1).sendMessage(message,
-                                                 withPushNotification: true,
-                                                 bundleIdForPushNotification: app.bundleId())
+        checkHappyPath(message: message, sender: sender)
+    }
+    
+    func test_pushNotification_optionalValuesEqualToNil() throws {
+        linkToScenario(withId: 27)
+
+        mockPushNotification(body: message)
+        
+        GIVEN("user goes to message list") {
+            userRobot.login().openChannel()
         }
-        THEN("user receives a push notification") {
-            userRobot.assertPushNotification(withText: message, from: sender)
+        checkHappyPath(message: message, sender: app.label.uppercased())
+    }
+    
+    func test_pushNotification_optionalValuesAreEmpty() throws {
+        linkToScenario(withId: 293)
+
+        mockPushNotification(body: message,
+                             title: "",
+                             badge: 0,
+                             mutableContent: 0,
+                             category: "",
+                             type: "",
+                             sender: "",
+                             version: "",
+                             messageId: "",
+                             cid: "")
```

Comment:
Usually we follow this format approach in the rest of the codebase, but not a big deal for now 👍 
```suggestion
        mockPushNotification(body: message,
                 title: "",
                 badge: 0,
                 mutableContent: 0,
                 category: "",
                 type: "",
                 sender: "",
                 version: "",
                 messageId: "",
                 cid: ""
         )
```